forest/drivers/gridded_forecast.py

```diff
@@ -142,10 +146,24 @@ def image(self, state):
             })
         return data
 
+    @staticmethod
+    def rollbabyroll(lons, values):
+        if np.any(lons > 180.0):
+            shift_by = np.sum(lons > 180.0)
+            lons[lons > 180.0] -= 360.
+            lons = np.roll(lons, shift_by)
+            values = np.roll(values, shift_by, axis=1)
+        return lons, values
+
```

Comment:
This patch could be removed completely since we've identified the true culprit to be cartopy `.transform_points()` wrapping `+/- 360` longitudes into a single x location

Reply:
agreed, method removed.